test/ReactViews/Tools/ItemSearchTool/ItemSearchToolSpec.tsx

```diff
@@ -86,6 +86,9 @@ describe("ItemSearchTool", function () {
   it("initializes an describes the parameters when mounted", async function () {
     spyOn(itemSearchProvider, "initialize").and.callThrough();
     spyOn(itemSearchProvider, "describeParameters").and.callThrough();
+    // FIXME: the await triggers the warning
+    //   "Do not await the result of calling act(...) with sync logic, it is not a Promise."
+    //   but removing the await makes the test for describeParameters fail.
     await act(() => {
       rendered = render({
         item,
```

Comment:
Luckily in this case the ItemSearchTool component accepts an [afterLoad](https://github.com/TerriaJS/terriajs/blob/a04930a0527cd9085fbe62c11f0c22f33ca9255d/lib/ReactViews/Tools/ItemSearchTool/ItemSearchTool.tsx#L67) callback, so we can use it to wait on.
Something like this should work:

```ts
    let renderPromise: Promise<void> | undefined;
    act(() => {
      renderPromise = new Promise((resolve) => render({
        item,
        itemSearchProvider,
        viewState,
        afterLoad: resolve
      }));
    });
    await renderPromise;
```

Reply:
Added a commit with the suggestion and the tests now pass, thanks!